src/bin/main.rs

```diff
@@ -0,0 +1,60 @@
+extern crate dhall_mock;
+
+use anyhow::{anyhow, Error};
+use log::{debug, info, warn};
+use signal_hook::iterator::Signals;
+
+use std::sync::{Arc, RwLock};
+use tokio::sync::mpsc::channel;
+
+use dhall_mock::cli;
+use dhall_mock::{compiler_executor, create_loader_runtime, run_web_server, start_logger, State};
+
+fn main() -> Result<(), Error> {
+    start_logger();
+
+    let cli_args = cli::load_cli_args();
+
+    let loading_rt = create_loader_runtime()?;
+    let web_rt = tokio::runtime::Runtime::new()?;
+
+    info!("Start dhall mock project 👋");
+    let (mut tx, rx) = channel(10);
+    let state = Arc::new(RwLock::new(State {
+        expectations: vec![],
+    }));
+
+    // Start dhall configuration loader
+    loading_rt.spawn(compiler_executor(rx, state.clone()));
+
+    let http_bind = cli_args.http_bind.clone();
+
+    // Send configuration to load from cli
+    loading_rt.spawn(async move {
```

Comment:
Is it possible to move all this part to the `lib.rs` module?

Reply:
Done :+1: